src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -9,11 +9,13 @@ import OptionsList from '../OptionsList';
 import CONST from '../../CONST';
 import styles from '../../styles/styles';
 import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import withNavigationFocus from '../withNavigationFocus';
```

Comment:
withNavigationFocusPropTypes is missing

Reply:
@Santhosh-Sellavel I am not sure if we need to add withNavigationFocusPropTypes, checked on other pages where we use withNavigationFocus and we are not adding withNavigationFocusPropTypes but right now I added withNavigationFocusPropTypes the propTypes, and have not added a default value for is focused as useIsFocused always returns the boolean value from withNavigationFocus HOC